helm/vitess/values.yaml

```diff
@@ -215,11 +215,11 @@ pmm:
       # It is recommended to set this limit to roughly 2/3 of the total memory that you are planning to allow for Prometheus.
       metricsMemory: "600000"
 
-# Orchestrator requires at least Kubernetes 1.9 to work
+# Orchestrator requires at least version >= 3.0.7 and Kubernetes 1.9 to work
 # Default values for orchestrator resources
 orchestrator:
   enabled: false
-  image: "vitess/orchestrator:3.0.6"
+  image: "vitess/orchestrator:3.0.7"
```

Comment:
Can you update the k8s/orchestrator Dockerfile to build 3.0.7?

Reply:
Done

Reply:
`3.0.8` is coming up very shortly with bugfixes to `3.0.7`.

Reply:
This should change to `3.0.9` now?